tools/lerno/README.md

```diff
@@ -0,0 +1,37 @@
+lerno
+=====
+
+A caching build tool for NodeJS monorepos.
+
+## Usage
+
+Run this in the root of the repository to do the build, quicker:
```

Comment:
Can this only run for the entire repo or can we run it for individual modules as well? (also built-in buildup/down functionality will be awesome)